### PR TITLE
Style/abd welcome page

### LINKED ISSUE #37: Convert all css files and imports to use scss

### PR DESCRIPTION
**Pull request by Abdeboskey**
#### What's this PR do?
* Adjust styling on welcome page based on group feedback:
  * Layout is now horizontal, no more scroll
  * Lightened background image and letter color, added invisible white background shadow behind main text. 
#### Where should the reviewer start?
* Begin at Instructions.scss, and peek at GameHistory.scss if ya wanna
#### How should this be manually tested?
* With the eyes 👀
#### Any background context you want to provide?
* I had to move the bookshelf component to the render of the instructions component so that I could control it in the correct container. 
* Some adjustments might need to be made once we have more than two books rendering on the bookshelf, but they shouldn't be too tricky. 
#### What are the relevant tickets?
#43 and closes #37
#### Screenshots (if appropriate)
Screen Shot 2020-09-14 at 8 02 27 PM
#### Questions:
Any other thoughts about this?

### CODE REVIEW

File: src/components/App/App.scss [hunk 1]
```diff
@@ -3,11 +3,13 @@
 
 .App {
   font-family: $main-font;
+  color: $graphite;
 }
 
 .background {
   width: 100%;
   position: fixed;
+  filter: contrast(90%);
```
Comment (carlymclift) [inline src/components/App/App.scss:13]: Really like what you did with the filter

File: src/components/GameHistory/GameHistory.scss [hunk 2]
```diff
@@ -1,21 +1,24 @@
+@import '../../styles/fonts.scss';
+
 .game-history-box {
-  position: fixed;
+  position: relative;
```
Comment (carlymclift) [inline src/components/GameHistory/GameHistory.scss:4]: Good work on fixed the positioning of this

**Review by carlymclift — APPROVED** [review]
Looks great Aaron!

File: src/components/App/App.scss (hunk 1 above)
Comment (GreysonElkins) [reply src/components/App/App.scss:13]: Yasssss

**Review by GreysonElkins — APPROVED** [review]
Thanks for hopping on with me to get those changes in!

File: src/components/App/App.scss [hunk 3]
```diff
@@ -4,6 +4,7 @@
 .App {
   font-family: $main-font;
   color: $graphite;
+  text-shadow: 0px 0px 1.5px $graphite;
```
Comment (nickhartdev) [inline src/components/App/App.scss:7]: LOVE the effect that this creates, super cool stuff

**Review by nickhartdev — APPROVED** [review]
Looks great!! Thanks for doing this

**Pull request merged**